frontend/src/styles/colors.ts

```diff
@@ -1,9 +1,9 @@
 // Colors names are taken from https://chir.ag/projects/name-that-color/
 
-const red = '#FF0000';
 const white = '#FFFFFF';
+const cornflowerBlue = '#7467EF';
```

Comment:
Nie wspominałem w sumie wcześniej, ale te nazwy kolorów sortujmy po nazwach alfabetycznie. Czyli defininicja `cornflowerBlue ` powinna być przed `white`.

Czyli to będzie tak:
`
const cornflowerBlue = '#7467EF';
const white = '#FFFFFF';
`